src/repositories/account.repo.js

```diff
@@ -0,0 +1,107 @@
+import { pool } from "../config/db.js";
+import { Account } from "../models/account.model.js";
+
+export class AccountRepo {
+  static async createAccount(account) {
+    const result = await pool.query(
+      `INSERT INTO accounts (user_name, user_email, user_password) VALUES ($1, $2, $3) RETURNING *`,
+      [account.user_name, account.user_email, account.user_password]
+    );
+    if (result.rows[0].length === 0) {
+      throw new Error("Unable to create new account");
+    }
+    console.log("repo: ", account);
```

Comment:
```suggestion
    console.log("[AccountRepo.createAccount]: ", account);
```

now your logs looks like 

repo: ............... when something goes wrong how you will know which function throw the error ?